src/FSharpVSPowerTools/Commands/PrintfSpecifiersUsageTaggerProvider.cs

```diff
@@ -85,20 +85,22 @@ public PrintfColorManager(ThemeManager themeManager, IEditorFormatMapService edi
         {
             _themeManager = themeManager;
             _editorFormatMapService = editorFormatMapService;
+
+            _currentTheme = _themeManager.GetCurrentTheme();
         }
 
         public Color GetDefaultColor()
         {
-            return _themeManager.GetCurrentTheme() == VisualStudioTheme.Dark ? DarkThemeColor : LightThemeColor;
+            return _currentTheme == VisualStudioTheme.Dark ? DarkThemeColor : LightThemeColor;
         }
 
         public void UpdateColors()
         {
-            var currentTheme = _themeManager.GetCurrentTheme();
+            var newTheme = _themeManager.GetCurrentTheme();
 
-            if (currentTheme != VisualStudioTheme.Unknown && currentTheme != lastTheme)
+            if (newTheme != VisualStudioTheme.Unknown && newTheme != _currentTheme)
```

Comment:
This logic is duplicated in `SyntaxConstructClassifierProvider`. Maybe move it to `VSColorTheme` so that `ThemeChanged` event is not raised if theme has not actually changed?